cli/integration_tests/basic_monorepo/dry_run.t

```diff
@@ -0,0 +1,97 @@
+    Cached (Local)         = false                                                                                             
+    Cached (Remote)        = false                                                                                             
+    Directory              = apps/my-app                                                                                       
+    Command                = echo 'building'                                                                                   
+    Outputs                =                                                                                                   
+    Log File               = apps/my-app/.turbo/turbo-build.log                                                                
+    Dependencies           =                                                                                                   
+    Dependendents          =                                                                                                   
+    ResolvedTaskDefinition = {"outputs":[],"cache":true,"dependsOn":[],"inputs":[],"outputMode":0,"env":[],"persistent":false} 
+  $ ${TURBO} run build --dry | grep "util#build" -A 12
+  util#build
+    Task                   = build                                                                                             
+    Package                = util                                                                                              
+    Hash                   = 238262ce1218fd3a                                                                                  
+    Cached (Local)         = false                                                                                             
+    Cached (Remote)        = false                                                                                             
+    Directory              = packages/util                                                                                     
+    Command                = echo 'building'                                                                                   
+    Outputs                =                                                                                                   
+    Log File               = packages/util/.turbo/turbo-build.log                                                              
+    Dependencies           =                                                                                                   
+    Dependendents          =                                                                                                   
+    ResolvedTaskDefinition = {"outputs":[],"cache":true,"dependsOn":[],"inputs":[],"outputMode":0,"env":[],"persistent":false} 
+
+# Validate output of my-app#build task
+  $ ${TURBO} run build --dry=json | jq '.tasks' | jq 'map(select(.taskId == "my-app#build"))' | jq '.[0]'
```

Comment:
```suggestion
  $ ${TURBO} run build --dry=json | jq '.tasks | map(select(.taskId == "my-app#build")) | .[0]'
```
You could also use `.tasks | .[] | select(.taskId == ...)` if you want to omit the `.[0]` at the end, but that would just be a style choice.